iron-doc-viewer.html

```diff
@@ -56,7 +57,10 @@
 
     <nav id="api">
       <header>API Reference</header>
-      <a id="togglePrivate" on-tap="_togglePrivate">{{_privateToggleLabel}}</a>
+      <paper-button id="togglePrivate"
+        tabindex="0"
```

Comment:
paper-button will automatically get a tabindex of 0 if you don't specify a tabindex, so you can leave this off.